.golangci.yml

```diff
@@ -1,19 +1,36 @@
 linters:
-  enable:
-    - gofmt
-    - govet
-    - errcheck
-    - staticcheck
-    - gosimple
-    - ineffassign
-    - typecheck
-    - unused
-    - misspell
-    - gocyclo
-    - gosec
-    - revive
+  enable-all: true
+  disable:
+    - exportloopref # Deprecated
+    - varnamelen # Not relevant
+    - exhaustruct # Not relevant
+    - ireturn # Not relevant
+    - nlreturn # Not relevant
+    - contextcheck # Too strict
+    - err113 # Too strict
+    - forcetypeassert # Too strict
+    - tagliatelle # Too strict
+    - wrapcheck # Too strict
+    - wsl # Too strict
+    - noctx # Too strict
```

Comment:
Not sure what this one does but I don't have it disabled, I assume it's also similar to contextcheck and should be enabled if it never complained for me

Reply:
This one assures that we use the context object in http call or in database operations. I'll remove it from disable list